dispatch_dynamic_analysis

tccontre opened the pull request:
### Details
- [ ]  playbooks/Dispatch_Dynamic_Analysis.json
- [ ]  playbooks/Dispatch_Dynamic_Analysis.png
- [ ]  playbooks/Dispatch_Dynamic_Analysis.py
- [ ]  playbooks/Dispatch_Dynamic_Analysis.yml
_What does this PR have in it? Screenshots are worth 1000 words 😄_

### Checklist

- [ ] Validate name matches `<platform>_<mitre att&ck technique>_<short description>` nomenclature
- [ ] [CI/CD](https://github.com/splunk/security_content/actions) jobs passed ✔️ 
- [ ] Validated SPL logic.
- [ ] Validated tags, description, and how to implement.
- [ ] Verified references match analytic.

tccontre committed:
dispatch_dynamic_analysis

srv-rr-gh-researchbt committed:
Branch was auto-updated.

srv-rr-gh-researchbt committed:
Branch was auto-updated.

P4T12ICK commented:
# Pull Request Type

##### Please check all that apply:
- [x] New playbook
- [ ] Bugfix
- [ ] Feature add
- [ ] Code style update (formatting, renaming)
- [ ] Documentation
- [ ] Other (please describe): 


# Playbook quality checklist
Please check if your PR fulfills the following requirements.

### Requirements for Settings

- [x] Playbook name is A-Z in Title case with underscores between words. (e.g. MS_Graph_Search_and_Purge)
- [x] Category in Title case with spaces between words (e.g. Identifier Reputation Analysis)
- [x] Description is free of grammatical errors and describe what the playbook does.
- [x] Notes list any setup required on the third-party API as well as intended areas for customization.
- [x] Label is set to '\*'
    
### Requirements for all playbooks

- [x] Playbook block count not greater than 15 (not including Start and End blocks).
- [x] No more than 3 branching paths.
- [x] If referencing a custom list, Notes document what the expected values are in that custom list.

### Requirements for all playbook blocks

- [x] All blocks have a custom name no more than 4 words, all lowercase, and separated by space (e.g. close workbook task)
- [x] All blocks that support a Notes Tooltip have it filled out. Must be grammatically correct and describes the intended purpose of that block.
- [x] Where custom code is used, block notes indicate presence of custom code (e.g. "This block uses custom code")
- [x] No block is disabled by custom code
- [x] Custom code is documented with notes

### Requirements for specific blocks

##### Action

- [x] Use apps available on Splunkbase
- [x] Use asset names that are the app name, all lowercase separated by underscores (e.g. Azure AD Graph becomes azure_ad_graph)

##### Utility 

- [x] Block is using community version

##### Playbook

- [x] Block is using local version

### Requirements for specific playbooks

##### Input playbooks
- [x] Start blocks use ocsf variable names and a minimum of one data type per variable name (e.g. device (type: host name))
- [x] Has at least one category tag (e.g. reputation)
- [x] Playbook has a tag for each vendor app used

# Other considerations (PR type specific)
- [x] If new playbook, there is a screenshot ending in .png with the same name as the playbook .json
- [x] Playbook major minor version matches repo (e.g. 5.5 != 6.0)
- [x] PR contains both .py and .json

---
Thanks for contributing!

tccontre committed:
dispatch_dynamic_analysis

P4T12ICK commented:
Looks good to me

srv-rr-gh-researchbt committed:
Branch was auto-updated.

srv-rr-gh-researchbt committed:
Branch was auto-updated.

srv-rr-gh-researchbt committed:
Branch was auto-updated.

srv-rr-gh-researchbt committed:
Branch was auto-updated.

srv-rr-gh-researchbt committed:
Branch was auto-updated.

srv-rr-gh-researchbt committed:
Branch was auto-updated.

srv-rr-gh-researchbt committed:
Branch was auto-updated.

srv-rr-gh-researchbt committed:
Branch was auto-updated.

tccontre committed:
dispatch_dynamic_analysis

srv-rr-gh-researchbt committed:
Branch was auto-updated.

srv-rr-gh-researchbt committed:
Branch was auto-updated.

srv-rr-gh-researchbt committed:
Branch was auto-updated.

srv-rr-gh-researchbt committed:
Branch was auto-updated.

srv-rr-gh-researchbt committed:
Branch was auto-updated.

srv-rr-gh-researchbt committed:
Branch was auto-updated.

srv-rr-gh-researchbt committed:
Branch was auto-updated.

srv-rr-gh-researchbt committed:
Branch was auto-updated.

srv-rr-gh-researchbt committed:
Branch was auto-updated.

srv-rr-gh-researchbt committed:
Branch was auto-updated.

srv-rr-gh-researchbt committed:
Branch was auto-updated.

srv-rr-gh-researchbt committed:
Branch was auto-updated.

srv-rr-gh-researchbt committed:
Branch was auto-updated.

srv-rr-gh-researchbt committed:
Branch was auto-updated.

srv-rr-gh-researchbt committed:
Branch was auto-updated.

srv-rr-gh-researchbt committed:
Branch was auto-updated.

srv-rr-gh-researchbt committed:
Branch was auto-updated.

srv-rr-gh-researchbt committed:
Branch was auto-updated.

kelby-shelton committed:
Rename playbooks and minor grammatical changes in .yml

kelby-shelton committed:
Made required changes. See description.

Playbook is failing when only some reputation results were returned.

These are the changes I made, since we don't want to delay this by waiting for another attack range:
1. Added a filter block after "filter successful observable output."
2. Adding the filter negates the need to do a custom code block before the format. Removed custom code.
3. Rename "filter successful observable output" to "observable output decision," because its not actually filtering. Its just a decision.
4. Reword "observable output of detonation playbooks not exits" to "No observable output exists for dispatched playbooks."
5. Switched beginning decision to filter and routed the filtered artifact IDs to the playbook.

kelby-shelton committed:
Fixed reference to different playbook.

srv-rr-gh-researchbt committed:
Branch was auto-updated.

srv-rr-gh-researchbt committed:
Branch was auto-updated.

srv-rr-gh-researchbt committed:
Branch was auto-updated.

srv-rr-gh-researchbt committed:
Branch was auto-updated.
